<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>431</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QLabel" name="l_range">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Область значений функции&lt;/p&gt;&lt;p&gt;Значение:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="2" column="1">
       <widget class="QSpinBox" name="b_pen">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="g_pen">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="r_pen">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>G</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="l_style">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Стиль линии</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="1">
       <widget class="QSpinBox" name="r_back">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>G</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="g_back">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="b_back">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="l_pen">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Цвет пера</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="l_width">
      <property name="text">
       <string>Ширина линии</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="back_l">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Цвет области</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QDial" name="range_r">
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="value">
       <number>40</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSpinBox" name="s_width">
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="save_settings">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить настройки</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
